### ISSUE TITLE
more control of which day or range you specifically want to backfill from

### ISSUE THREAD

**Issue by geekpete** [wakatara/harsh#2]
We're trying to keep harsh as minimal as possible, but...

**Is your feature request related to a problem? Please describe.**
Just started using the tool so there's no history yet.
If I go to fill in my log using ask, it starts from the furthest unfilled date but I only want to fill in yesterday's log.

If you start with an empty log it does ask you about how far back you want to start filling the log in from:

```
$ h a 
Your log file looks empty. Let's setup your tracking.
How many days back shall we start tracking from in days?
harsh will ask you about each habit for every day back.
Starting today would be 0. Choose. (0-7) 
2
2020-06-23:
```

but the next time you go to log, it goes back to the furthest point to start asking for data:

```
$ h a                                                                                                                                                                                                           
2020-06-17:

```

**Describe the solution you'd like**
I can just go through and skip all these to get up to current or where I want to get up to, but it might be nice to be able to pick the data I want to fill in specifically or from which date forward I want to fill in for.

**Describe alternatives you've considered**
I guess I could go hand adjust the log file manually.

**Additional context**

**Comment by wakatara** [comment]
@geekpete 

Ugh. Yes, I see what it's doing. I probably need to pick up a flag for the earliest date the logs have now for each habit that I introduced that feature. Unintended side effect of the added onboarding feautures. Thanks for the report. Will look at this in next day or so and try to push out a fix (I also need to push snaps packaging for linux users as well.).

thanks!
Daryl.

**Comment by wakatara** [comment]
And just so I'm crystal on this @geekpete , what you want to be able to do is specify the ISO date of where the logging should start from in setting up?

**Comment by wakatara** [comment]
Hey @geekpete 

As described, where the app went back before your first date should now be corrected as of 0.8.5 and 0.8.6. We've also got snaps fer y'all if you're using linux and are keen on snaps.

Have a great weekend!
